doc/update_all_files.sh

```diff
@@ -9,8 +9,8 @@ DOC_DIR=`dirname $0`
 BASE_DIR=`pwd $DOC_DIR/..`
 
 # Update source files
-SOURCE_FILES=`find $BASE_DIR -type f \( -name *.cc -or -name *.h \) -and -not -name *.bak | grep -v doc`
-bash ${DOC_DIR}/update_source_files.sh $SOURCE_FILES
+#SOURCE_FILES=`find $BASE_DIR -type f \( -name *.cc -or -name *.h \) -and -not -name *.bak | grep -v doc`
```

Comment:
what is the reason for this change?

Reply:
That was a leftover of my updates, I fixed it in a second commit.